.github/workflows/test-docker.yml

```diff
@@ -65,7 +80,7 @@ jobs:
           docker exec -u ghrunner hdk-build.${{ inputs.name }} tar -zxf /_work/build.tgz -C /_work/
 
       - name: Sanity test
-        if: inputs.name == 'cuda'
+        if: inputs.name == 'cuda'|| inputs.name == 'clang-cpu'
```

Comment:
Seems like this didn't run for the clang-built HDK, huh?

Reply:
yep